payments-core/src/main/java/com/stripe/android/networking/StripeRequest.kt

```diff
@@ -9,7 +10,8 @@ import java.util.HashSet
 /**
  * A class representing a request to a Stripe-owned service.
  */
-internal abstract class StripeRequest {
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+abstract class StripeRequest {
```

Comment:
Same above re: auditing what we're making public now

Reply:
Done, `internal`ized the properties